worlds/tunic/docs/en_TUNIC.md

```diff
@@ -67,4 +67,22 @@ For the Entrance Randomizer:
 Bombs, consumables (non-bomb ones), weapons, melee weapons (stick and sword), keys, hexagons, offerings, hero relics, cards, golden treasures, money, pages, and abilities (the three ability pages). There are also a few groups being used for singular items: laurels, orb, dagger, magic rod, holy cross, prayer, icebolt, and progressive sword.
 
 ## What location groups are there?
-Holy cross (for all holy cross checks), fairies (for the two fairy checks), well (for the coin well checks), and shop. Additionally, for checks that do not fall into the above categories, the name of the region is the name of the location group.
\ No newline at end of file
+Holy cross (for all holy cross checks), fairies (for the two fairy checks), well (for the coin well checks), and shop. Additionally, for checks that do not fall into the above categories, the name of the region is the name of the location group.
+
+## Is Connection Plando supported?
+Yes. The host needs to enable it in their `host.yaml`, and the player's yaml needs to contain a plando_connections block.
+Example:
+```
+plando_connections:
+  - entrance: Stick House Entrance
+    exit: Stick House Exit
+  - entrance: Special Shop Exit
+    exit: Stairs to Top of the Mountain
+```
+Notes:
+- The Entrance Randomizer option must be enabled for it to work.
+- The `direction` field is not supported. Connections are always coupled.
+- For a list of entrance names, check `er_data.py` in the TUNIC world folder or generate a game with the Entrance Randomizer option enabled and check the spoiler log.
+- There is no limit to the number of Shops hard-coded into place.
+- If you have more than one shop in a scene, you may be wrong warped when exiting a shop.
+- If you have a shop in every scene, and you have an odd number of shops, it will error out.
```

Comment:
What's wrong with an odd number of shops?

Reply:
If you have a shop in every scene, and it tries to place another shop, it'll fail because it avoids trying to place a shop in a scene that already has one (because I don't want to wrong warp the player). However, if there is an even number of shops, since there's an even number of non-shop portals already, it won't actually try to place any more shops, and therefore will not go through that failure path.